### PR TITLE
fix(kubernetes/helm): point out intended use for this stage

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

**Review by skim1420 — APPROVED** [review]

File: guides/user/kubernetes-v2/deploy-helm/index.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ with the help of a templating engine. Currently, the only supported templating
 engine is [Helm](https://helm.sh/), by relying on the `helm template` command.
 See more details [here](https://docs.helm.sh/helm/#helm-template).
 
+> Note: This stage is intended to help you package and deploy applications 
+> that you own, and are actively developing and redeploying frequently.
+> It is not intended to serve as a one-time installation method for
+> third-party packages. If that is your goal, it's arguably better to call
+> `helm apply` once when bootstrapping your Kubernetes cluster.
+
```
Comment (dorbin) [inline guides/user/kubernetes-v2/deploy-helm/index.md]: @lwander Is there anything you could link to for more information?

**Review by dorbin — APPROVED** [review]
One minor question, but lgtm otherwise

File: guides/user/kubernetes-v2/deploy-helm/index.md (hunk 1 above)
Comment (lwander) [reply guides/user/kubernetes-v2/deploy-helm/index.md]: More information about what?

Comment (dorbin) [reply guides/user/kubernetes-v2/deploy-helm/index.md]: I was thinking `helm apply`, but I don't find it on [helm.sh](https://helm.sh/).

Comment (lwander) [reply guides/user/kubernetes-v2/deploy-helm/index.md]: Good call, I was even pointing out the wrong command for the version of helm we support

**Pull request merged**